plugins/inputs/sqlserver/sqlserverqueries.go

```diff
@@ -292,8 +292,8 @@ END
 DECLARE
 	 @SqlStatement AS nvarchar(max)
 	,@MajorMinorVersion AS int = CAST(PARSENAME(CAST(SERVERPROPERTY('ProductVersion') AS nvarchar),4) AS int)*100 + CAST(PARSENAME(CAST(SERVERPROPERTY('ProductVersion') AS nvarchar),3) AS int)
-	,@Columns AS nvarchar(MAX) = ''
-	,@PivotColumns AS nvarchar(MAX) = ''
+	,@Columns AS nvarchar(MAX) = '
+	,@PivotColumns AS nvarchar(MAX) = '
```

Comment:
looks unintentional

Reply:
You are right, I replaced some single quotes and that introduced that error, as it should be 2 single-quotes.
Anyway turns out those variables are no longer needed and can be removed, as there were only used in the shythetic counters section.

Thanks for spotting it.

Reply:
I already made a commit to remove the variables